chromadb/segment/impl/metadata/sqlite.py

```diff
@@ -252,7 +252,6 @@ def _record(self, rows: Sequence[Tuple[Any, ...]]) -> MetadataEmbeddingRecord:
 
         return MetadataEmbeddingRecord(
             id=embedding_id,
-            seq_id=_decode_seq_id(seq_id),
```

Comment:
@HammadB, does it make sense also to drop seq_id from the table and queries altogether?

Reply:
we can, in the future, didn't want to make a migration on the main data tables